mmv1/third_party/validator/bigquery_dataset_iam.go

```diff
@@ -34,15 +34,15 @@ func resourceConverterBigqueryDatasetIamMember() ResourceConverter {
 	}
 }
 
-func GetBigqueryDatasetIamPolicyCaiObject(d TerraformResourceData, config *transport_tpg.Config) ([]Asset, error) {
+func GetBigqueryDatasetIamPolicyCaiObject(d tpgresource.TerraformResourceData, config *transport_tpg.Config) ([]Asset, error) {
```

Comment:
you'll need to add this import at the top of all of the handwritten files that need it.

Reply:
Done.